File: pattern/19.cpp
#include <iostream>
using namespace std;

int main()
{
    int rows = 5;
    int n = 5;
    int h = 1;
    for (int i = 0; i < rows; i++)
    {
        for (int j = 0; j < n; j++)
        {
            cout << "*";
        }
        int temp = n;
        n--;
        for (int k = 0; k < (h - 1) * 2; k++)
        {
            cout << " ";
        }
        h++;
        for (int k = 0; k < temp; k++)
        {
            cout << "*";
        }

        cout << endl;
    }

    int rowss = 5;
    int nn = 1;
    int hh = 5;
    for (int ii = 0; ii < rowss; ii++)
    {
        for (int jj = 0; jj < nn; jj++)
        {
            cout << "*";
        }
        int tempp = nn;
        nn++;
        for (int kk = 0; kk < (hh - 1) * 2; kk++)
        {
            cout << " ";
        }
        hh--;
        for (int kk = 0; kk < tempp; kk++)
        {
            cout << "*";
        }

        cout << endl;
    }

    return 0;
}

File: pattern/19.py
rows = 5
n = 5
h = 1

# First pattern
for i in range(rows):
    # Print first set of stars
    for j in range(n):
        print("*", end="")
    temp = n
    n -= 1
    # Print spaces
    for k in range((h - 1) * 2):
        print(" ", end="")
    h += 1
    # Print second set of stars
    for k in range(temp):
        print("*", end="")
    print()  # Move to the next line

rowss = 5
nn = 1
hh = 5

# Second pattern
for ii in range(rowss):
    # Print first set of stars
    for jj in range(nn):
        print("*", end="")
    tempp = nn
    nn += 1
    # Print spaces
    for kk in range((hh - 1) * 2):
        print(" ", end="")
    hh -= 1
    # Print second set of stars
    for kk in range(tempp):
        print("*", end="")
    print()  # Move to the next line
